pom/flight_pom.py

```diff
@@ -1,72 +1,55 @@
 
-    data_ = XlData()
-    variable1_, keys_ = data_.read_locators(file_name, sheet_name)
-    ObjectGen_ = GenericMethod()
+class FlightRegister():
 
     def __init__(self, driver):
         self.driver = driver
+        self.variable1_ = ReadJson.read_locators(OBJECT_JSON)
+        self.ObjectGen_ = GenericMethod()
 
     def pop_up_ok(self):
-        reg_pop = FlightRegister.variable1_["pop_up_ok"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg_pop)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["pop_up_ok"])
 
     def round_trip(self):
-        reg3 = FlightRegister.variable1_["round_trip"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg3)
-        return FlightRegister.ObjectGen_.validate_element_display(self.driver, FlightRegister.variable1_["return_date_click"])
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["round_trip"])
+        return self.ObjectGen_.validate_element_display(self.driver,self.variable1_["return_date_click"])
 
     def station_from_sending_keys(self):
-        reg4 = FlightRegister.variable1_["station_from_sending_keys"]
-        FlightRegister.ObjectGen_.enter_text(self.driver, reg4, values="Chennai")
+        self.ObjectGen_.enter_text(self.driver, self.variable1_["station_from_sending_keys"], values="Chennai")
 
     def station_from_click(self):
-        reg5 = FlightRegister.variable1_["station_from_click"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg5)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["station_from_click"])
 
     def station_to_sending_keys(self):
-        reg6 = FlightRegister.variable1_["station_to_sending_keys"]
-        FlightRegister.ObjectGen_.enter_text(self.driver, reg6, values="Mumbai")
+        self.ObjectGen_.enter_text(self.driver, self.variable1_["station_to_sending_keys"], values="Mumbai")
 
     def station_to_click(self):
-        reg7 = FlightRegister.variable1_["station_to_click"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg7)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["station_to_click"])
 
     def start_date_select(self):
-        reg8 = FlightRegister.variable1_["start_date_select"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg8)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["start_date_select"])
 
     def return_date_click(self):
-        reg9 = FlightRegister.variable1_["return_date_click"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg9)
+       self.ObjectGen_.click_on_element(self.driver, self.variable1_["return_date_click"])
 
     def apirl_month_select(self):
-        reg10 = FlightRegister.variable1_["apirl_month_select"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg10)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["apirl_month_select"])

```

Comment:
Make start_date_select() and apirl_month_select() into one method, which takes date as input parameter.

Reply:
this i merged with below two ,return date click and return date select